test/data/valid.tf

```diff
@@ -16,28 +16,19 @@ resource "instaclustr_cluster" "valid" {
         name = "AWS_VPC"
     }
     rack_allocation = {
-        number_of_racks = 3
-        nodes_per_rack = 1
+        number_of_racks = 5
+        nodes_per_rack = 2
     }
 
     bundle {
         bundle = "APACHE_CASSANDRA"
-        version = "3.11.4"
+        version = "apache-cassandra-3.11.8.ic2"
         options = {
             auth_n_authz = true
-            use_private_broadcast_rpc_address = true
-            lucene_enabled = true
+            use_private_broadcast_rpc_address = false
+            client_encryption = false
+            lucene_enabled = false
             continuous_backup_enabled = true
         }
     }
-
-    bundle {
-        bundle = "SPARK"
-        version = "apache-spark:2.3.2"
-    }
-
-    bundle {
-        bundle = "ZEPPELIN"
-        version = "apache-zeppelin:0.8.0-spark-2.3.2"
-    }
```

Comment:
[Required] I think spark should be included but zeppelin can be removed since its EOL in terms of our support of for it

Reply:
done